src/main/java/study/lotto/controller/LottoController.java

```diff
@@ -30,27 +26,24 @@ public void start() {
     }
 
     private void showResult(DrawResult drawResult, LottoPurchaseHistory lottoPurchaseHistory) {
-        view.showWinningStatictics(WinningStatistics.of(drawResult, lottoPurchaseHistory.getTotalCost()));
+        view.showWinningStatictics(new WinningStatistics(drawResult, lottoPurchaseHistory));
     }
 
     private DrawResult draw(Lottos purchasedLotto) {
-        Lotto winningLotto = Lotto.from(view.getWinningLottoNumbers().get());
-        LottoDraw lottoDraw = new LottoDraw(winningLotto, view.getBonusBall().get());
+        LottoDraw lottoDraw = new LottoDraw(view.getWinningLotto(), view.getBonusBall().get());
```

Comment:
```suggestion
        LottoDraw lottoDraw = new LottoDraw(view.getWinningLotto(), view.getBonusBall());
```

get을 한번 더 해주신 이유가 있을까요?

Reply:
아 !! 리팩토링에서 놓친 부분이네요. 😳 

변경하도록 하겠습니다!